test/monitor/start-stop-test.js

```diff
@@ -52,7 +53,13 @@ vows.describe('forever-monitor/monitor/start-stop').addBatch({
               child.once("restart", function(){
                 child.stop()
               })
-              child.once("exit", function(){ that.callback(null, child) })
+              child.once("exit", function(){
+                // wait another two seconds, to make sure the child is not restarted after calling stop()
+                setTimeout(function() {
+                  clearTimeout(timer)
```

Comment:
This test had an additional issue, which was that it wasn't clearing the above timer on success, nor was it declaring `timer` as a `var`.